src/DataMiner/SLManagedAutomation/UIResults.cs

```diff
@@ -198,6 +198,68 @@ public string[] GetExpanded(string key)
 			return new string[0];
 		}
 
+		/// <summary>
+		/// Gets the filter value from the specified destination variable that is linked to a dialog box item.
+		/// </summary>
+		/// <param name="key">The name of the destination variable.</param>
+		/// <returns>The filter value from the specified destination variable that is linked to a dialog box item.</returns>
+		/// <example>
+		/// <code>
+		/// UIResults results;
+		/// string initialValue = null;
+		/// string filter = null;
+		/// 
+		/// do
+		/// {
+		///   var uib = new UIBuilder()
+		///   {
+		///     RequireResponse = true,
+		///     RowDefs = "a;a",
+		///     ColumnDefs = "a",
+		///   };
+		/// 
+		///   var dropDownControl = new UIBlockDefinition
+		///   {
+		///     InitialValue = initialValue,
+		///     Type = UIBlockType.DropDown,
+		///     Row = 0,
+		///     Column = 0,
+		///     DestVar = "FilterDropdown",
+		///     WantsOnFilter = true,
+		///   };
+		///   uib.AppendBlock(dropDownControl);
+		///   // Add options to dropDownControl based on filter
+		/// 
+		///   var closeButton = new UIBlockDefinition
+		///   {
+		///     Type = UIBlockType.Button,
+		///     Text = "Close",
+		///     Row = 1,
+		///     Column = 0,
+		///     DestVar = "Close",
+		///   };
+		///   uib.AppendBlock(closeButton);
+		/// 
+		///   results = engine.ShowUI(uib);
+		/// 
+		///   if (results.WasOnFilter("FilterDropdown"))
+		///   {
+		///     initialValue = results.GetString("FilterDropdown");
+		///     filter = results.GetFilterString("FilterDropdown");
+		///   }
+		/// }
+		/// while (!results.WasButtonPressed("Close"));
+		/// </code>
+		/// </example>
+		/// <remarks>
+		/// <para>Applicable only in case <see cref="Type"/> is set to <see cref="UIBlockType.DropDown"/>.</para>
+		/// <para>For <see cref="GetFilterString"/> to work, <see cref="UIBlockDefinition.WantsOnFilter"/> has to be set to true and <see cref="WasOnFilter"/> for the current result should return true.</para>
+		/// <note type="note">Available from DataMiner 10.5.8/10.6.0 onwards, in Automation scripts launched from web apps and <see href="xref:Configuring_app_actions_and_behavior_via_URL#configuring-app-behavior-via-the-url">specifying the useNewIASInputComponents=true URL parameter</see>.</note> <!-- RN 42808 / RN 42845 -->
+		/// <note type="important">When the options are filtered, the currently selected option (if still relevant) needs to be inserted as an option, regardless of whether the filter matches. Otherwise, the dropdown will consider that value incorrect, and the dropdown will be cleared.</note>
+		/// <note type="tip">Consider only filtering the display value of the options and, if possible, case invariant.</note>
```

Comment:
Could you clarify how "case invariant" is linked grammatically to the rest of the sentence? Something seems off about the sentence, but I'm not sure how best to adjust it. (same for line 498)

Reply:
The string GetFilterString() returns, is used to filter out a list of options display in the list of a dropdown. Considering 'fra' as the filter string, my suggestion would be to also consider 'France' as a correct option. That won't be the case when using String.Contains in C# by default.

Reply:
Would this be an acceptable alternative?
```suggestion
		/// <note type="tip">Consider only filtering the display value of the options and, if possible, ensure that the filter is not case-sensitive.</note>
```